app/src/main/java/com/github/sdpcoachme/messaging/InAppNotificationService.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.sdpcoachme.messaging
+
+import android.annotation.SuppressLint
+import android.content.ContentValues
+import android.util.Log
+import com.github.sdpcoachme.CoachMeApplication
+import com.github.sdpcoachme.database.Database
+import com.google.android.gms.tasks.OnCompleteListener
+import com.google.firebase.messaging.FirebaseMessaging
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+
+/**
+ * This service handles all incoming push notifications.
```

Comment:
It could be good to have a better description of this class and the difference between it and InAppNotifier. For now, both class description are pretty similar and it is hard to tell what they do without looking at the code

Reply:
Great point, I will adapt the class description